### DISCUSSION TITLE
How to add system installed wine?

### DISCUSSION THREAD

**Discussion by Mart-Bogdan** [DavidoTek/ProtonUp-Qt#473] [Q&A]
I've figured out that game that I want works fine with system installed wine (wine-9.16) but I want features of steam as screenshots. 

But this tool only allows me to Download new version of wine, and not regular, but wine-GE. Would be nice if this was an option.

**Comment by sonic2kk** [comment]
System-installed Wine should be managed by your package manager and does not natively integrate with any game launchers. ProtonUp-Qt lists tools installed into folders for use by game launchers, not those just generally available on your system.

If you would like to use vanilla Wine with a game, you can use the Kron4ek Wine builds.
